### ISSUE TITLE
ROG Ally power button doesn't work

### ISSUE THREAD

**Issue by superm1** [CachyOS/CachyOS-Handheld#35]
Currently `HandlePowerKey=ignore` is set in https://github.com/CachyOS/CachyOS-Handheld/blob/main/etc/systemd/logind.conf.d/steam-deckify.conf

On ROG Ally there is currently no powerbutton service installed and so this stops power button from working. 

There are two options to solve this issue I can see:
1. Start installing https://github.com/ShadowBlip/steam-powerbuttond with CachyOS handheld. This generally provides a "better" experience because Steam client will suspend.
2. Modify `steam-deckify.conf` when running on a non-Steam deck to set `HandlePowerKey=suspend`

**Comment by superm1** [comment]
I just ran updates recently on the Ally and (not surprisingly) my changes to `/etc/systemd/logind.conf.d/steam-deckify.conf` were wiped.

**Comment by ptr1337** [comment]
> I just ran updates recently on the Ally and (not surprisingly) my changes to `/etc/systemd/logind.conf.d/steam-deckify.conf` were wiped.

Likely because this is not packaged. I will add it tomorrow.

**Comment by superm1** [comment]
> Likely because this is not packaged. I will add it tomorrow.

While doing that can it be split up as profiles?  I think it should _only_ be ignored when deck is found, right?

**Comment by ptr1337** [comment]
> > Likely because this is not packaged. I will add it tomorrow.
> 
> While doing that can it be split up as profiles? I think it should _only_ be ignored when deck is found, right?

Yes, we will likely handle it via chwd for new installations. For existing ones I need to check, what is the best way to do so.

**Comment by superm1** [comment]
I don't mind some sort of manual clean up process to get on the right track, so if it's best to do it that way I'm totally fine.  I just want to make sure it is right for people in the future :)

**Comment by vnepogodin** [comment]
maybe we can set HandlePowerKey via DBUS on condition

**Comment by ventureoo** [comment]
We should move the settings here (except maybe `/etc/skel` and` /etc/xdg`) to `/usr`, like we did in cachyos-settings, so that users have the ability to override configuration files.
